src/assets/styles/base.scss

```diff
@@ -267,6 +267,19 @@ ul.list {
   color: $primary-blue;
 }
 
+// General icon styling
+.header-icon {
+  size: 25px !important;
+  color: $app-dk-blue !important;
+  vertical-align: top !important;
+}
+
+.info-icon {
+  size: 22px !important;
+  color: $app-blue !important;
+  vertical-align: top !important;
```

Comment:
Is "important" needed for all properties? I think it's better to reserve the use of "important" to components that need to override it -- so base properties should minimize the use of "important".
(It is still possible to override these properties if needed in the future by being more specific with the CSS selector, or perhaps by "latest match wins" logic.)

Reply:
FWIW, I have not seen any guidance or best practises on CSS in base classes vs components, so my opinion is based on my experience only.

Reply:
Leave it as is. I trust your experience and intuition here.